src/__tests__/fire-event.js

```diff
@@ -215,6 +227,31 @@ test('fireEvent.update does not trigger warning messages', async () => {
   expect(console.warn).not.toHaveBeenCalled()
 })
 
+test('fireEvent.update should not crash with input file', async () => {
```

Comment:
Is this test example not enough? @afontcu

Reply:
It is! But as mentioned, I wouldn't make `fireEvent.update()` smarter when it comes to type="file" inputs. I would just trigger the proper event, and then add a test showcasing how to handle files as in React Testing Library (and probably also a comment suggesting userEvent to do so:


```js
test('fireEvent.update should not crash with input file', async () => {
  const {getByTestId} = render({
    template: `<input type="file" data-testid="test-update" />`
  })

  const file = new File(['(⌐□_□)'], 'chucknorris.png', { type: 'image/png' })

  const inputEl = getByTestId('test-update')

  Object.defineProperty(inputEl, 'files', {
    value: [file],
  })

  await fireEvent.update(inputEl)

  expect(console.warn).not.toHaveBeenCalled()
})
```

Reply:
I added a suggestion with the change I mention above 👍